.github/workflows/kotlin-lint.yml

```diff
@@ -44,9 +44,9 @@ jobs:
           path: |
             ~/.gradle/caches
             ~/.gradle/wrapper
-            android/build
-            android/.gradle
-          key: ${{ runner.os }}-kotlin-lint-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties', 'android/build.gradle') }}
+            packages/react-native-gesture-handler/android/build
```

Comment:
Can `packages/react-native-gesture-handler/` be extracted somewhat? maybe an env variable?

Reply:
Done in [7b585ac](https://github.com/software-mansion/react-native-gesture-handler/pull/3497/commits/7b585aceb0b2729af9df90faf067de6d36c7b9d4)